core/src/main/scala/chisel3/Data.scala

```diff
@@ -665,6 +665,12 @@ abstract class Data extends HasId with NamedComponent with SourceInfoDoc {
    */
   def litValue: BigInt = litOption.get
 
+  /**
+    * Returns the value as BitInt
+    * @return
+    */
+  def toBigInt: BigInt = litOption.get
```

Comment:
I personally think this is a bad API to directly get the `litOption`.
What if the user `toBigInt` a `Reg`/`Wire`/`Node` in their Chisel design?
And this is just a function alias to `litValue`.
I do think this is a good sugar, but I think if we are going to add this, it should remain in `chiseltest` rather than chisel3.